python/packages/sdk/sls_sdk/base.py

```diff
@@ -3,15 +3,14 @@
 from datetime import datetime
 from typing import List, Union
 
-from importlib_metadata import version
 from typing_extensions import Final
 
 
 SLS_ORG_ID: Final[str] = "SLS_ORG_ID"
 
 # module metadata
 __name__ = "serverless-sdk"
-__version__: Final[str] = version(__name__)
```

Comment:
Isn't that change problematic maintenance wise? Will we now have to bump this version manually every time? We definitely should not do that